Rawl_B502/_94.trig

```diff
@@ -418,7 +418,7 @@
     a foaf:Person;
     irishRel:genName "Dedaid";
     irishRel:nomName "Dedad";
-    owl:sameAs <http://example.com/Rawl_B502/mínigud_senchais_ébir_inso.trig#Dedaid>.
+    owl:sameAs <http://example.com/Duanaire_Finn/PoemXXXVII.trig#Deadhadh>.
```

Comment:
I had a look at this and seems like this shouldn't change based on the names and the relationships.  I just want to make sure I understand what this change does.  :)

Reply:
You're right. Looking at all three, `<http://example.com/Rawl_B502/mínigud_senchais_ébir_inso.trig#Dedaid>` should be the same as `<http://example.com/Duanaire_Finn/PoemXXXVII.trig#Deadhadh>` and `<http://example.com/Rawl_B502/_94#Dedad>`. However, no further change is needed, as they are all linked up via another instance of the same person anyway.